update_version.yml

```
@@ -0,0 +1,29 @@
+name: Update Version on Docker Files
+
+env:
+  PRJ_VER: ${{ github.event.inputs.project_version }}
```

Can we have a longer name here?
```suggestion
  PROJECT_VERSION: ${{ github.event.inputs.project_version }}
```